changelog.d/20220407_145118_benc_executor_batch_default.rst

```diff
@@ -0,0 +1,10 @@
+Changed
+^^^^^^^
+
+- FuncXExecutor will use batched submission by default. This can lead to significant
+  improvements in task submission rate when using the executor interface (for example,
+  3 seconds to submit 500 tasks vs 2 minutes, in an informal test). However,
+  individual task submission latency may be increased.
+
+  To use non-batched submission mode, set batch_mode=False when creating FuncXExecutor.
```

Comment:
Suggest text formatting the `code` and linking to the (most relevant) documentation

```suggestion
- `FuncXExecutor <https://funcx.readthedocs.io/en/latest/executor.html>`_
  now uses batched submission by default.  This typically significantly
  improves the task submission rate when using the executor interface (for
  example, 3 seconds to submit 500 tasks vs 2 minutes, in an informal test).
  However, individual task submission latency may be increased.
 
  To use non-batched submission mode, set `batch_mode=False` when instantiating
  the `FuncXExecutor <https://funcx.readthedocs.io/en/latest/executor.html>`_
  object.
```